facet-core/src/impls_camino.rs

```diff
@@ -57,7 +57,7 @@ unsafe impl Facet<'_> for Utf8PathBuf {
             ))
             .vtable(
                 &const {
-                    let mut vtable = value_vtable!((), |f, _opts| write!(f, "Utf8PathBuf"));
+                    let mut vtable = value_vtable!(String, |f, _opts| write!(f, "Utf8PathBuf"));
```

Comment:
Any reason this can't just be `Utf8PathBuf`? I think this is what it should've been from the beginning.

(Same remark for `Utf8PathBuf` below)

Reply:
Good question! As I mentioned, I have no idea what I'm doing, so I tried the first thing that came to mind, and then it didn't even cross my mind that I could use the actual type (I thought this was some kind of "base this on that type that we already know about"). Thanks for setting me straight :) Fixed in https://github.com/facet-rs/facet/pull/481/commits/3f08c50c877ac38c2dd1ab43fb357856f3e4dcb4